front/src/applications/stdcmV2/components/StdcmConfig.tsx

```diff
@@ -71,24 +94,15 @@ const StdcmConfig = ({
     <div className="stdcm-v2__body">
       <div className="stdcm-v2-simulation-settings">
         <div className="stdcm-v2-consist-container">
-          <StdcmConsist
-            disabled={isPending}
-            setCurrentSimulationInputs={setCurrentSimulationInputs}
-          />
+          <StdcmConsist {...inputsProps} />
         </div>
         <div className="stdcm-v2__separator" />
         <div className="stdcm-v2-simulation-itinerary">
           {/* //TODO: use them when we implement this feature #403 */}
           {/* <StdcmDefaultCard text="Indiquer le sillon antérieur" Icon={<ArrowUp size="lg" />} /> */}
-          <StdcmOrigin
-            disabled={isPending}
-            setCurrentSimulationInputs={setCurrentSimulationInputs}
-          />
-          <StdcmVias disabled={isPending} setCurrentSimulationInputs={setCurrentSimulationInputs} />
-          <StdcmDestination
-            disabled={isPending}
-            setCurrentSimulationInputs={setCurrentSimulationInputs}
-          />
+          <StdcmOrigin {...inputsProps} />
+          <StdcmVias {...inputsProps} />
```

Comment:
I'm not sure if using `onChange` is the best approach to determine when to show or hide the button. Instead, we should base the decision on whether `selectedSimulation.inputs` is equal to `currentSimulationInputs` (the simulation being edited). Because, there are many where we can change the `currentSimulationInputs`